### ISSUE TITLE
when scan  EAN_13 barcode is  always not read

### ISSUE THREAD

**Issue by xinggg22** [ideacreation/react-native-barcodescanner#17]
When a EAN_13 barcode in a good's package ,the barcode may be small and have some font around  it, and then the sanner is  always not read...  :<

**Comment by jeremybviseo** [comment]
I agree with you, there is a problem with EAN_13 identification. All other EAN types work fine.

**Comment by diogoviannaaraujo** [comment]
Yeah, here in brazil most barcodes are EAN_13, really sad :(

**Comment by ThaJay** [comment]
haven't been able to scan either EAN_13 or QR

**Comment by ThaJay** [comment]
tried this yet? https://github.com/ideacreation/react-native-barcodescanner/issues/37#issuecomment-231708733

**Comment by Anuj-786** [comment]
Hey, Which version of react you are using. for me it scanned the Barcode EAN_13 successfully.

![ean_13](https://cloud.githubusercontent.com/assets/15275386/18908117/fedc4496-858b-11e6-99bb-2d2872c29e38.png)
